build.gradle

```diff
@@ -437,6 +437,25 @@ project('amqp') {
 
 }
 
+project('barrier') {
+	description = 'Barrier Sample'
+
+	apply plugin: 'spring-boot'
+
+	dependencies {
+		compile 'org.springframework.boot:spring-boot-starter-web'
+		compile 'org.springframework.boot:spring-boot-starter-integration'
+		compile "org.springframework.integration:spring-integration-core:4.2.0.BUILD-SNAPSHOT"
+		compile "org.springframework.integration:spring-integration-amqp:4.2.0.BUILD-SNAPSHOT"
+		compile "org.springframework.integration:spring-integration-http:4.2.0.BUILD-SNAPSHOT"
```

Comment:
`$springIntegrationVersion` is more feasible here.
Although I don't think that we should be in a hurry and there is just enough to wait for the proper Boot version to pick up those SI changes.
WDYT?